R/cal-apply-binary.R

```diff
@@ -64,7 +64,7 @@ cal_apply_binary.cal_estimate_beta <- function(object,
 
 #---------------------------- Adjust implementations ---------------------------
 
-cal_add_cls_predict_impl <- function(object, .data) {
+apply_binary_predict <- function(object, .data) {
```

Comment:
The code for using isotonic regression with a numeric outcome is 99% percent the same as this function. Can we rename it to be a little more general?

Reply:
or refactor it so that we can re-use most of it?